prompt_selection.ipynb

```diff
@@ -121,6 +122,153 @@
     "    'suffix': ['Please try to be as funny as possible.', '']})"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Experiment with mock llm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from response_checker import *\n",
+    "\n",
+    "class MockLLMChain:\n",
+    "    outcomes: List[List[float]] = None\n",
+    "    \n",
+    "    def __init__(self, outcomes, prompt):\n",
+    "        self.outcomes = outcomes\n",
+    "        self.prompt = prompt\n",
```

Comment:
maybe put mock checker in a test_utils file or something so that we can re-use?

Reply:
done